cmd/remove.go

```diff
@@ -17,18 +17,23 @@ Example:
   $ shorten remove go
   $ shorten remove x29jzI8m`,
 	Run: func(cmd *cobra.Command, args []string) {
-		repoName := args[0]
 
-		repo, err := findRepo(repoName)
-		if err != nil {
-			fmt.Printf("Repository `%s` not found.\n", repoName)
-			return
+		// do input check before running command
+		if len(args) == 0 {
+			var repoName string
+			fmt.Print("Please input shortened URL you want to remove : ")
```

Comment:
This should read:

`"Please input the name(s) of the repos you'd like to remove: "`